plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -3026,11 +3031,16 @@ public VmStatsEntry getVmStat(final Connect conn, final String vmName) throws Li
         try {
             dm = getDomain(conn, vmName);
```

Comment:
I believe here we can have a null pointer in very specific cases. For instance, if this code is being executed while a call to destroy the VM that has the name “vmName”; this getDomain would return a null value.